### ISSUE TITLE
BPE and wav2vec

### ISSUE THREAD

**Issue by abnerLing** [pytorch/fairseq#3177]
I was curious how we should organize the data when using BPE instead of letters for the dictionary like in the vq-wav2vec paper which used a 4K bpe output vocab.

Should the dictionary look like this? Is it okay to have more than 1 letter per line?
```
▁t      -0
▁th     -1
▁a      -2
▁i      -3
▁w      -4
▁the    -5
re      -6
▁s      -7
▁o      -8
in      -9
```
Also, should the train.wrd and train.ltr have all text encoded? For example, 
train.wrd
`▁feel ▁that ▁we ▁have`
train.ltr
`▁ f e e l | ▁ t h a t | ▁ w e | ▁ h a v e |`

**Comment by lematt1991** [comment]
Yep, this is the correct format for the dictionary.  Specifically, it looks like

```
<symbol0> <count0>
<symbol1> <count1>
...
```

Where `<symbol*>`  is the token, and `<count*>` is the number of times that it occurs, but if you don't care about doing anything with the counts (some tasks like MLM use it for weighted sampling), you can just put dummy values in.

**Comment by ArtemisZGL** [comment]
@lematt1991 hello,  I  found that there is the param 'wrd' when finetuning the wav2vec2, so i have tried the word level finetuning,  but the result seems much worse than the letter level, did i miss somethin?
